redwood-cli/src/main/kotlin/app/cash/redwood/generator/diffConsumingGeneration.kt

```diff
@@ -37,52 +36,40 @@ import com.squareup.kotlinpoet.TypeSpec
 import com.squareup.kotlinpoet.asTypeName
 
 /*
-public class DiffConsumingSunspotWidgetFactory<W : Any>(
-  private val delegate: SunspotWidgetFactory<W>,
+public class SunspotDiffConsumingNodeFactory<W : Any>(
+  private val widgets: SunspotWidgetFactory<W>,
   private val json: Json = Json.Default,
   private val mismatchHandler: ProtocolMismatchHandler = ProtocolMismatchHandler.Throwing,
 ) : DiffConsumingNode.Factory<W> {
-  override val RedwoodLayout = DiffConsumingRedwoodLayoutWidgetFactory(delegate.RedwoodLayout, json, mismatchHandler)
```

Comment:
Similarly, no need to generate a type to hold functions which `new` up a class. Inlined below.